tests/sources/test_sharepoint_online.py

```diff
@@ -1663,20 +1663,26 @@ async def test_get_attachment_with_text_extraction_enabled_adds_body(
         with patch(
             "connectors.utils.ExtractionService.extract_text", return_value=message
         ) as extraction_service_mock:
+            with patch(
```

Comment:
This file is easier to review if you hide whitespace changes.

Reply:
You can actually do both patches in one `with`: https://stackoverflow.com/a/73884949